{
  final AccountModelDao account=createTestAccount(1,"12345678901234567890123456");
  try {
    accountDao.create(account,internalCallContext);
    Assert.fail();
  }
 catch (  RuntimeException e) {
    Assert.assertTrue(e.getCause() instanceof SQLException);
  }
}
